### PR TITLE
Refactor/user params

### PR DESCRIPTION
**Pull request by EricBelongea**
## Description

Please include a description of what was implemented. What was changed? What was added or removed if anything? 

I needed to change the user params so that I am able to create a new user. The params were requiring :user which blocks the creation of a user. Can't have what we are creating? If there's another way I am happy to hear hear about it!

This breaks and Updates the create and update tests in user_request_spec 

## Type of change
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe testing implemented and what all it covers. 

- [ ]  fully tests happy and sad paths

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [x] all tests pass

### CODE REVIEW

**Comment by EricBelongea** [comment]
I changed the Create and Update tests in the user_request_spec. The tests failed when I changed the user_params in the UsersController from...

 "params.require(:user).permit(:name, :email, :password)" ---> "params.permit(:name, :email, :password)"

**Comment by JustinSteel** [comment]
looks good to me i like the changes

File: app/controllers/api/v1/users_controller.rb [hunk 1]
```diff
@@ -46,6 +46,6 @@ def set_user
 
     # Only allow a list of trusted parameters through.
     def user_params
-      params.require(:user).permit(:name, :email, :password)
+      params.permit(:name, :email, :password)
```
Comment (ttakahashi1591) [inline app/controllers/api/v1/users_controller.rb:49]: This update looks good!

File: spec/requests/api/v1/users_request_spec.rb [hunk 2]
```diff
@@ -60,9 +60,11 @@
...
                       password: "gw45635yhethet5"
                     })
-      headers = {"CONTENT_TYPE" => "application/json"}
-    
-      post api_v1_users_path, headers: headers, params: JSON.generate(user: user_params)
+      # headers = {"CONTENT_TYPE" => "application/json"}
+      # post api_v1_users_path, headers: headers, params: JSON.generate(user: user_params)
+
+      #Above (line:63-64) is what was here, and below ln:67 is the new suggestion `EB`
+      post api_v1_users_path, params: user_params
```
Comment (ttakahashi1591) [inline spec/requests/api/v1/users_request_spec.rb:67]: Could you elaborate on what exactly this difference is doing? I thought that the original syntax including the headers was necessary for the request to go through?

**Review by JustinSteel — APPROVED** [review]

**Pull request merged**